release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3143,6 +3143,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * There is no log pruning available for azcopy (underlying tool running copy jobs) in Azure, so this might eventually lead to filling up a root device of the controller pod, and you have to manually clean this up. (link:https://issues.redhat.com/browse/OCPBUGS-35980[*OCPBUGS-35980*])
 
+* The limited live migration method stops when the `ConfigMap` MTU object is missing and returns the following error: `NetworkTypeMigrationFailed. Failed to process SDN live migration (configmaps "mtu" not found)`.
++
+In most cases, the `ConfigMap` MTU object is created by the `mtu-prober` job during installation. However, if the cluster was upgraded from an early release, for example, {product-title} 4.4.4, the `ConfigMap` MTU object might be absent.
++
+As a temporary workaround, you can manually create the `ConfigMap` MTU object before starting the limited live migration process. (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
```

Comment:
Here's an example of the configmap. 
Note: the MTU value shall be aligned with the MTU of the node interface.
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: mtu
  namespace: openshift-network-operator
data:
  mtu: "1500"
```